plugins/core/IO/qPDALIO/src/LASFilter.cpp

```diff
@@ -362,9 +362,9 @@ CC_FILE_ERROR LASFilter::saveToFile(ccHObject* entity, const QString& filename,
 
 	//optimal scale (for accuracy) --> 1e-8 because the maximum integer is roughly +/-2e+9
```

Comment:
Should this comment be updated? I know it's not relevant to your change, just bringing it up

Reply:
Sorry - I missed this.

I don't know :-) Maybe @dgirardeau has thoughts on it?

I think I'll go ahead with the commit since it's been pending for so long and if something needs to be done with this comment, it can be done separately.

Reply:
Ah ah yes, it's probably 1e-9 instead of 1e-8, but that's not a big deal